### PR TITLE
Add boolean group_membership for whether to export contacts group mem…

### PR DESCRIPTION
**Pull request by norkans7**
…bership sheet

### CODE REVIEW

File: temba/contacts/views.py [hunk 1]
```diff
@@ -531,35 +541,63 @@ def render_to_response(self, context, **response_kwargs):
...
-                        self.request,
-                        _("Export complete, you can find it here: %s (production users will get an email)") % dl_url,
-                    )
+            previous_export = (
+                ExportContactsTask.objects.filter(org=org, created_by=user).order_by("-modified_on").first()
+            )
+            if previous_export and previous_export.created_on < timezone.now() - timedelta(
+                hours=24
+            ):  # pragma: needs cover
+                analytics.track(self.request.user.username, "temba.contact_exported")
```
Comment (nicpottier) [inline temba/contacts/views.py]: What is this doing? We are logging an event when there is a recent export?

Comment (norkans7) [reply temba/contacts/views.py]: we only log up to once a day

**Comment by norkans7** [comment]
@nicpottier Also I think this PR can wait to test the other approaches first

**Comment by norkans7** [comment]
Not needed anymore. replaced by https://github.com/nyaruka/rapidpro/pull/1962

**Pull request closed without merging**